appserver/pom.xml

```diff
@@ -59,6 +59,21 @@
       <url>https://github.com/javaee/glassfish</url>
     </scm>
     
+    <repositories>
+        <!-- TODO: remove me, I'm here for testing only-->
```

Comment:
whole repositories block will go away as soon as maven.java.net allows... Integration must be done using 'promoted' or 'releases' repositories which are already defined in parent pom